package.json

```diff
@@ -7,7 +7,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "@moltin/sdk": "github:ynnoj/js-sdk",
```

Comment:
This dependency looks are very old one and now it has no permission for me to run `npm install` or `yarn install`.
Not sure if the version is right with Next.js but using the latest one.